MoneyLoverClone/Scenes/Account/AccountViewController.swift

```diff
@@ -40,32 +42,83 @@ class AccountViewController: UIViewController {
             $0.dataSource = self
             $0.register(cellType: AccountTableViewCell.self)
         }
+        tableViewAccount.isScrollEnabled = false
+    }
+    func logout(){
```

Comment:
```suggestion
    func logout() {
```
cách dòng 46 bằng 1 khoảng trắng